### PR TITLE
feat(locale): de color human

### PR DESCRIPTION
**Pull request by xDivisionByZerox**
Add common, human-readable colors to the german locale.

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Review by matthewmayer — DISMISSED** [review]

File: src/locales/de/color/human.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  'Magenta',
+  'Mintgrün',
+  'Nachtblau',
+  'Neonblau',
+  'Neongelb',
+  'Neongrün',
+  'Neonrot',
+  'Ocker',
+  'Orange',
+  'Pastel',
```
Comment (Shinigami92) [inline src/locales/de/color/human.ts]: To me, pastel is a color category, not a color itself 🤔

Comment (xDivisionByZerox) [reply src/locales/de/color/human.ts]: <s>I thought so myself. But according to the color lists I used, it is a separate color^^</s>

You may be right. I remove it. Although I'm not able to do so until sunday

Comment (Shinigami92) [reply src/locales/de/color/human.ts]: You have a link? What color lists are you referring to?

Comment (xDivisionByZerox) [reply src/locales/de/color/human.ts]: The specific entry was listed here:
https://de.m.wiktionary.org/wiki/Verzeichnis:Deutsch/Farben#P

Comment (Shinigami92) [reply src/locales/de/color/human.ts]: 1. There it is called pastell with two l
2. Maybe use something like pastellgelb as like as neongeld

Comment (ST-DDT) [reply src/locales/de/color/human.ts]:
```suggestion
```

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**